README.md

```diff
@@ -37,24 +43,29 @@ npm i
 
 ## Setup
 
-To integrate with this test suite, you will need a
-[VC-API compatible](https://w3c-ccg.github.io/vc-api/) issuer and verifier that
-are capable of issuing and verifying verifiable credentials and verifiable
-presentations. If your implementation is not VC-API compatible, it is possible
-to "wrap" the implementation in a minimal VC-API implementation, example code
-for which is available at <https://github.com/Wind4Greg/Server-for-VCs>.
-Additionally, your verifier will need to be able to verify Verifiable Credentials
+This test suite uses an HTTP API (based on the
+[VC-API](https://w3c-ccg.github.io/vc-api/)) for passing  credentials into
+issuer and verifier implementations. If your implementation is not already
+compatible with the VC-API, it is possible to "wrap" the implementation in a
+minimal VC-API implementation (example code is available at
+<https://github.com/Wind4Greg/Server-for-VCs>).
+
+Additionally, verifier implementations will need to be able to verify Verifiable Credentials
 and Verifiable Presentations signed with `eddsa-rdfc-2022`. We recommend
-that any issuer endpoints submitted to this test suite also issue using
+that any issuer endpoints used with this test suite also issue using
 `eddsa-rdfc-2022`. Both signed and unsigned Verifiable Presentations will
 be submitted for verification. Signed Verifiable Presentations from this suite
 will have a domain and challenge set. The domain should be the test repo, and
 the challenge is static.
 
-The issuer endpoint will need to conform to the
-[VC Issuer API](https://w3c-ccg.github.io/vc-api/#issue-credential).
+### Test Suite HTTP API
 
-A request to issue a credential (`/credentials/issue`) will look like this:
+
+A request to **issue a credential** (`/credentials/issue`) will look like this:
+
+```http
+POST /credentials/issue
```

Comment:
shouldn't this contain the request body which will be (in most cases)

```
{
  "credential": {...},
  "options": {...}
}
```

Reply:
The request body is immediately below this on line 69, fwiw.